src/main/java/com/redhat/coolstore/model/Order.java

```diff
@@ -1,136 +1,130 @@
-package com.redhat.coolstore.model;
+// Update the `Order` class to use `jakarta.persistence`
+import io.quarkus.hibernate.orm.jpa.PersistenceConfiguration;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import jakarta.persistence.Transient;
 
 import java.io.Serializable;
 import java.util.ArrayList;
 import java.util.List;
 
-import javax.persistence.CascadeType;
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.FetchType;
-import javax.persistence.GeneratedValue;
-import javax.persistence.Id;
-import javax.persistence.JoinColumn;
-import javax.persistence.OneToMany;
-import javax.persistence.Table;
+import javax.validation.constraints.Digits;
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
 
 @Entity
 @Table(name = "ORDERS")
 public class Order implements Serializable {
 
-	private static final long serialVersionUID = -1L;
+    private static final long serialVersionUID = -1L;
 
-	@Id
-	@GeneratedValue
-	private long orderId;
+    @Id
+    @GeneratedValue
+    private long orderId;
 
-	private String customerName;
+    private String customerName;
 
-	private String customerEmail;
+    private String customerEmail;
 
-	private double orderValue;
+    private double orderValue;
 
-	private double retailPrice;
+    private double retailPrice;
 
-	private double discount;
+    private double discount;
 
-	private double shippingFee;
+    private double shippingFee;
 
-	private double shippingDiscount;
+    private double shippingDiscount;
 
-	@Column(name="TOTAL_PRICE")
+    @Column(name="TOTAL_PRICE")
 
-	
-	@OneToMany(fetch = FetchType.EAGER, cascade = CascadeType.ALL, orphanRemoval = true)
-	@JoinColumn(name="ORDER_ID")
-	private List<OrderItem> itemList = new ArrayList<>();
+    @OneToMany(fetch = FetchType.EAGER, cascade = CascadeType.ALL, orphanRemoval = true)
+    @JoinColumn(name="ORDER_ID")
+    private List<OrderItem> itemList = new ArrayList<>();
 
-	public Order() {}
+    // Add @Transient annotation to remove the `discount` field from the `toArray()` method
+    @Transient
+    private double discountField;
```

Comment:
Lots of meddling with these attributes, which doesn't seem to make a lot of sense